crates/shared/src/sources/uniswap_v3/pool_fetching.rs

```diff
@@ -0,0 +1,390 @@
+use super::graph_api::{PoolData, Token, UniV3SubgraphClient};
```

Comment:
Changes from this file are already reviewed here: https://github.com/cowprotocol/cow-native-liquidity/blob/master/liquidity-sources/src/sources/uniswap_v3/pool_fetching.rs